docs/data/toolpad/core/components/sign-in-page/sign-in-page.md

```diff
@@ -44,15 +44,86 @@ The following providers are supported and maintained by default:
 - Twitch
 - Discord
 - Keycloak
-- Credentials (username/password)
 
 Find details on how to set up each provider in the [Auth.js documentation](https://authjs.dev/getting-started/authentication/oauth).
 :::
 
+## Magic Link
```

Comment:
Should there be a default success alert that would already show in this demo?

Reply:
We have a separate demo on the Alerts just below this one, so I think this one is okay? If you think strongly that we should merge them, I can do that